vault-admin-client/src/main/java/com/scality/osis/vaultadmin/impl/VaultAdminImpl.java

```diff
@@ -217,7 +227,7 @@ public String getAccountsMarker(int offset, String filterKey) throws VaultServic
             cacheListAccountsMarker(listAccountsResponse.getAccounts().size(), marker);
 
           } else{
-            throw new VaultServiceException(400, "Requested offset is outside the total available items");
+            throw new VaultServiceException(HttpStatus.BAD_REQUEST, "Requested offset is outside the total available items");
```

Comment:
Just a suggestion, but I am okay with any.

You can use, InvalidParameterValue: https://github.com/scality/Arsenal/blob/development/8.1/errors/arsenalErrors.json#L426

Reply:
I am ok to change this but anyways in the end, OSIS can return only 200 for `list Accounts`